homeassistant/components/landisgyr_heat_meter/manifest.json

```diff
@@ -7,7 +7,5 @@
   "documentation": "https://www.home-assistant.io/integrations/landisgyr_heat_meter",
   "homekit": {},
   "iot_class": "local_polling",
-  "requirements": ["ultraheat-api==0.5.1"],
-  "ssdp": [],
-  "zeroconf": []
+  "requirements": ["ultraheat-api==0.5.1"]
```

Comment:
This should be a separate cleanup PR (and probably cleanup `homekit` section also...)

Reply:
Thanks, removed from PR (and will create a new one later).